packages/react/test/specs/components/Dropdown/Dropdown-test.tsx

```diff
@@ -57,7 +58,7 @@ describe('Dropdown', () => {
       )
     })
 
-    it('calls onOpen with highlightedIndex + 1 on arrow down', () => {
+    it('has the provided prop value + 1 when opened by ArrowDown', () => {
```

Comment:
just for the sake of better clarity, lets use braces:

> has value of (provided prop value + 1) when opened by ArrowDown

Reply:
sure, good idea!